public/js/pimcore/element/helpers/gridColumnConfig.js

```diff
@@ -406,64 +406,79 @@ pimcore.element.helpers.gridColumnConfig = {
 
         const fieldInfo = this.grid.getColumns()[columnIndex].config;
 
+        let editor;
+        let tagType;
         if (this.objecttype === "object" || this.objecttype === "variant") {
             if (!fieldInfo.layout || !fieldInfo.layout.layout) {
                 return;
             }
 
-            const tagType = fieldInfo.layout.type;
-            const editor = new pimcore.object.tags[tagType](null, fieldInfo.layout.layout);
+            tagType = fieldInfo.layout.type;
+            editor = new pimcore.object.tags[tagType](null, fieldInfo.layout.layout);
             editor.setObject(this.object);
-            editor.updateContext({
-                containerType: "filterByRelationWindow"
-            });
+        } else {
```

Comment:
I know it is different in other places, but should we do here an `else if (this.gridType === 'asset')` just to be sure?

Reply:
should I have an else clause that raises some error, etc?

Reply:
would not do that, no - as it could be seen as a BC break (very unlikely someone uses that, but still...)

Reply:
I am not sure that would be any more breaking. if you do if / else approach and neither branch is hit, we will still get an exception on `editor.updateContext({` as editor won't have been set

Reply:
hm, but that wasn't the case before, right? 

maybe we need an else clause with a return statement?

Reply:
@fashxp I wanted to circle back on this remaining question.

Reply:
sorry, did't had the check the bigger context ... will do asap.

Reply:
so, now I had the chance to check: that is what I meant with 'I know it is different in other places' in my first comment. I know it would be broken on other places, but that doesn't mean we cannot make new parts more robust ;-)

I would do an else clause with a return statement - that should keep existing behavior for 'wrong' usages, right?

Reply:
the return statement in the else clause does look to be safe / an improvement. I will push